### PR TITLE
Initialize Service translations in Bahasa Indonesia.

### PR DESCRIPTION
**Pull request by irvifa**
Initialize translations for Service
This address #13929

### CODE REVIEW

File: content/id/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -0,0 +1,1067 @@
...
+alternatif penggunaan `Service` untuk HTTP/HTTPS.
+{{< /note >}}
+
+### Protokol PROXY
+
+{{< feature-state for_k8s_version="v1.1" state="stable" >}}
+
+Apabila penyedia layanan `cloud` yang kamu gunakan mendukung, (misalnya saja, [AWS](/docs/concepts/cluster-administration/cloud-providers/#aws)),
+`Service` dengan `type` `LoadBalancer` untuk melakukan konfigurasi `load balancer` 
+diluar Kubernetes sendiri, serta akan melakukan `forwarding` koneksi yang memiliki prefiks
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: nit: di luar

File: content/id/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -0,0 +1,1067 @@
...
+
+Mulai `Services`.
+
+Sebuah `Service` pada Kubernetes adalah sebuah abstraksi yang memberikan definisi 
+set logis yang terdiri beberapa `Pod` serta `policy` bagaimana cara kamu mengakses set `Pod` tadi - seringkali disebut sebagai `microservices`.
+Set `Pod` yang dirujuk oleh suatu `Service` (biasanya) ditentukan oleh sebuah [`Label Selector`](/docs/concepts/overview/working-with-objects/labels/#label-selectors) 
+(lihat penjelasan di bawah untuk mengetahui alasan kenapa kamu mungkin saja membutuhkan `Service` tanpa 
+sebuah `selector`).
+
+Sebagai contoh, misalnya terdapat sebuah `backend` yang menyediakan fungsionalitas `image-processing`
```
[the remark this answers is not in the document]
Comment (irvifa) [reply content/id/docs/concepts/services-networking/service.md]: Agreed..

File: content/id/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -0,0 +1,1067 @@
...
+        # value. Defaults to 5, must be between 2 and 60
+
+        service.beta.kubernetes.io/aws-load-balancer-extra-security-groups: "sg-53fae93f,sg-42efd82e"
+        # A list of additional security groups to be added to ELB
+```
+
+#### Network Load Balancer support on AWS [alpha]
+
+{{< warning >}}
+Ini merupakan tingkatan alfa dan tidak direkomendasikan untuk digunakan pada `environment` `production`.
```
[the remark this answers is not in the document]
Comment (irvifa) [reply content/id/docs/concepts/services-networking/service.md]: alpha

File: content/id/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -0,0 +1,1065 @@
...
+
+### Tanpa _selector_
+
+Untuk _Service_ _"headless"_ tanpa  _selector_, kontroler _Endpoints_ 
+tidak akan membuat _record_ _Enpoints_. Meskipun demikian, 
+sistem DNS tetap melakukan konfigurasi salah satu dari:
+
+  * _record_ CNAME untuk [_ExternalName_](#externalname)-tipe services.
+  * _record_ untuk semua _Endpoints_ yang memiliki nama _Service_ yang sama, untuk 
+    untuk tipe lainnya.
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: repeated "untuk"

File: content/id/docs/concepts/services-networking/service.md [hunk 5]
```diff
@@ -0,0 +1,1065 @@
...
+
+## Mekanisme _publish_ _Service_ - jenis-jenis _Service_
+
+Untuk beberapa bagian dari aplikasi yang kamu miliki (misalnya saja, _frontend_), 
+bisa saja kamu memiliki kebutuhan untuk mengekspos _Service_ yang kamu miliki 
+ke alamat IP eksternal (di luar kluster Kubernetes).
+
+_ServiceTypes_ yang ada pada Kubernetes memungkinkan kamu untuk menentukan 
+jenis _Service_ apakah yang kamu butuhkan. Secara _default_,jenis _Service_
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: missing space after comma

File: content/id/docs/concepts/services-networking/service.md [hunk 6]
```diff
@@ -0,0 +1,1065 @@
...
+yang diberikan adalah _ClusterIP_.
+
+_Value_ dan perilaku dari tipe _Service_ dijelaskan sebagai berikut:
+
+   * _ClusterIP_: Mengekspos _Service_ ke _range_ alamat IP di dalam kluster. Apabila kamu memilih _value_ ini 
+     _Service_ yang kamu miliki hanya dapat diakses secara internal. tipe ini adalah 
+     _default_ _value_ dari _ServiceType_.
+   * [_NodePort_](#nodeport): Mengekspos _Service_ pada setiap IP *node* pada _port_ statis 
+     atau _port_ yang sama. Sebuah _Service_ _ClusterIP_, yang mana _Service_ _NodePort_ akan di-_route_
+     akan dibuat secara otomatis. Kamu dapat mengakses _Service_ dengan tipe ini,
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: repeated "akan"

...akan di-route akan...

File: content/id/docs/concepts/services-networking/service.md [hunk 7]
```diff
@@ -0,0 +1,1065 @@
...
+     dari luar kluster melalui `<NodeIP>:<NodePort>`.
+   * [_LoadBalancer_](#loadbalancer): Mengekspos _Service_ secara eksternal dengan menggunakan _LoadBalancer_ 
+     yang disediakan oleh penyedia layanan _cloud_. _Service_ dengan tipe _NodePort_ dan _ClusterIP_, 
+     dimana trafik akan di-_route_, akan dibuat secara otomatis.
+   * [_ExternalName_](#externalname): Melakukan pemetaan _Service_ ke konten 
+      dari _field_ _externalName_ (misalnya: _foo.bar.example.com_), dengan cara mengembalikan 
+      catatan _CNAME_ beserta _value_-nya. Tidak ada metode _proxy_ apa pun yang diaktifkan. Mekanisme ini 
+      setidaknya membutuhkan _kube-dns_ versi 1.7.
+
+### Type NodePort {#nodeport}
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: fields and flags should all be in backtick, follow the format according https://deploy-preview-14117--kubernetes-io-master-staging.netlify.com/docs/concepts/services-networking/service/#nodeport

File: content/id/docs/concepts/services-networking/service.md [hunk 8]
```diff
@@ -0,0 +1,1065 @@
...
+    metadata:
+      name: my-service
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
+```
+
+Sejak versi 1.3.0, penggunaan anotasi berlaku untuk semua _port_ yang diproksi oleh ELB 
+dan tidak dapat diatur sebaliknya.
+
+#### Akses _Log_ ELB pada AWS
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: annotation values should all be in backtick, follow https://deploy-preview-14117--kubernetes-io-master-staging.netlify.com/docs/concepts/services-networking/service/#elb-access-logs-on-aws

File: content/id/docs/concepts/services-networking/service.md [hunk 9]
```diff
@@ -0,0 +1,1065 @@
...
+        service.beta.kubernetes.io/aws-load-balancer-healthcheck-timeout: "5"
+        # The amount of time, in seconds, during which no response means a failed
+        # health check. This value must be less than the service.beta.kubernetes.io/aws-load-balancer-healthcheck-interval
+        # value. Defaults to 5, must be between 2 and 60
+
+        service.beta.kubernetes.io/aws-load-balancer-extra-security-groups: "sg-53fae93f,sg-42efd82e"
+        # A list of additional security groups to be added to ELB
+```
+
+#### Network Load Balancer support on AWS [alpha]
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: translate support on AWS

File: content/id/docs/concepts/services-networking/service.md [hunk 10]
```diff
@@ -0,0 +1,1065 @@
+---
+title: Services
+feature:
+  title: Service discovery and load balancing
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: Service discovery dan load balancing

File: content/id/docs/concepts/services-networking/service.md [hunk 11]
```diff
@@ -0,0 +1,1065 @@
+---
+title: Services
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: Prefer Service over Services

File: content/id/docs/concepts/services-networking/service.md [hunk 12]
```diff
@@ -0,0 +1,1065 @@
...
+{{< /note >}}
+
+![Ikhtisar diagram _Services_ pada _proxy_ _ipvs_](/images/docs/services-ipvs-overview.svg)
+
+Dari sekian model _proxy_ yang ada, trafik _inbound_ apa pun yang ada diterima oleh _IP:Port_ pada `Service` 
+akan dilanjutkan melalui _proxy_ pada _backend_ yang sesuai, dan klien tidak perlu mengetahui 
+apa informasi mendetail soal Kubernetes, `Service`, atau `Pod`. afinitas _session_ (_session affinity_) berbasis 
+_Client-IP_ dapat dipilih dengan cara menerapkan nilai _"ClientIP"_ pada `service.spec.sessionAffinity` 
+(nilai _default_ untuk hal ini adalah _"None"_), kamu juga dapat mengatur nilai maximum _session_ 
+_timeout_ yang ada dengan mengatur opsi `service.spec.sessionAffinityConfig.clientIP.timeoutSeconds`_ jika
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: typo: trailing underscore after `...timoutSeconds`

File: content/id/docs/concepts/services-networking/service.md [hunk 13]
```diff
@@ -0,0 +1,1065 @@
...
+
+## Memilih sendiri alamat IP yang kamu inginkan
+
+Kamu dapat memberikan spesifikasi alamat _cluster IP_ yang kamu inginkan 
+sebagai bagian dari _request_ pembuatan objek `Service`. Untuk melakukan hal ini, 
+kamu harus mengisi _fields_ `.spec.clusterIP` field. Contoh penggunaannya adalah sebagai berikut, 
+misalnya saja kamu sudah memiliki _entry_ DNS yang ingin kamu gunakan kembali, 
+atau sebuah sistem _legacy_ yang sudah diatur pada alamat IP spesifik 
+dan sulit untuk diubah. Alamat IP yang ingin digunakan pengguna haruslah merupakan alamat IP 
+yang valid dan berada di dalam _range_ _CIDR_ `service-cluster-ip-range`_ yang dispesifikasikan di dalam
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: trailing underscore after `service-cluster-ip-range`

File: content/id/docs/concepts/services-networking/service.md [hunk 14]
```diff
@@ -0,0 +1,1065 @@
...
+
+Mulai versi 1.9, `Service` juga dapat menggunakan [_predefined_ _policy_](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html)
+untuk HTTPS atau _listener_ SSL. Untuk melihat _policy_ apa saja yang dapat digunakan, kamu dapat menjalankan perintah _awscli_:
+
+```bash
+aws elb describe-load-balancer-policies --query 'PolicyDescriptions[].PolicyName'
+```
+
+_Policy_ ini kemudian dapat dispesifikasikan menggunakan anotasi 
+"_service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy_", contohnya:
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy need to be in backtick

File: content/id/docs/concepts/services-networking/service.md [hunk 15]
```diff
@@ -0,0 +1,1065 @@
...
+
+#### Dukungan _Network Load Balancer_ (NLB) pada AWS [alpha]
+
+{{< warning >}}
+Ini merupakan tingkatan _alpha_ dan tidak direkomendasikan untuk digunakan pada _environment_ _production_.
+{{< /warning >}}
+
+Sejak versi 1.9.0, Kubernetes mendukung _Network Load Balancer_ (NLB). Untuk
+menggunakan NLB pada AWS, gunakan anotasi `service.beta.kubernetes.io/aws-load-balancer-type`
+dan atur _value_-nya dengan _nlb_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: nit: nlb needs to be in backtick

File: content/id/docs/concepts/services-networking/service.md [hunk 16]
```diff
@@ -0,0 +1,1065 @@
...
+_security group_ dari _node_ akan diberikan _rules_ IP sebagai berikut:
+
+| _Rule_ | Protokol | `Port` | _IpRange(s)_ | Deskripsi _IpRange_ |
+|--------|----------|--------|--------------|---------------------|
+| _Health Check_   | TCP  | NodePort(s) (`.spec.healthCheckNodePort` for _.spec.externalTrafficPolicy = Local_) | VPC CIDR | kubernetes.io/rule/nlb/health=\<loadBalancerName\> |
+| _Client Traffic_ | TCP  | NodePort(s) | `.spec.loadBalancerSourceRanges` (defaults to _0.0.0.0/0_) | kubernetes.io/rule/nlb/client=\<loadBalancerName\> |
+| _MTU Discovery_  | ICMP | 3,4 | `.spec.loadBalancerSourceRanges` (defaults to _0.0.0.0/0_) | kubernetes.io/rule/nlb/mtu=\<loadBalancerName\> |
+
+Perhatikan bahwa jika `.spec.loadBalancerSourceRanges` tidak dispesifikasikan, 
+Kubernetes akan mengizinkan trafik dari _0.0.0.0/0_ ke_Node Security Group_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: nit: need space

ke _Node Security Group_

Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: 0.0.0.0/0 needs to be in backtick

File: content/id/docs/concepts/services-networking/service.md [hunk 17]
```diff
@@ -0,0 +1,1065 @@
...
+| _Client Traffic_ | TCP  | NodePort(s) | `.spec.loadBalancerSourceRanges` (defaults to _0.0.0.0/0_) | kubernetes.io/rule/nlb/client=\<loadBalancerName\> |
+| _MTU Discovery_  | ICMP | 3,4 | `.spec.loadBalancerSourceRanges` (defaults to _0.0.0.0/0_) | kubernetes.io/rule/nlb/mtu=\<loadBalancerName\> |
+
+Perhatikan bahwa jika `.spec.loadBalancerSourceRanges` tidak dispesifikasikan, 
+Kubernetes akan mengizinkan trafik dari _0.0.0.0/0_ ke_Node Security Group_. 
+Jika _node_ memiliki akses publik, maka kamu harus memperhatikan tersebut karena trafik yang tidak berasal 
+dari NLB juga dapat mengakses semua _instance_ di _security group_ tersebut.
+
+Untuk membatasi klien IP mana yang dapat mengakses NLB, 
+kamu harus memberikan spesifikasi _loadBalancerSourceRanges_.
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: loadBalancerSourceRanges needs to be in backtick

File: content/id/docs/concepts/services-networking/service.md [hunk 18]
```diff
@@ -0,0 +1,1065 @@
...
+yang kemudian diikuti data dari klien.
+
+### SCTP
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Kubernetes memberikan dukungan bagi SCTP sebagai _value_ dari _definition_ yang ada pada 
+_Service_, `Endpoints`, _NetworkPolicy_ dan `Pod` sebagai fitur _alpha_. Untuk mengaktifkan fitur ini, 
+administrator kluster harus menaktifkan _feature gate_ _SCTPSupport_ pada _apiserver_, contohnya 
+_“--feature-gates=SCTPSupport=true,...”_. Ketika _fature gate_ ini diaktifkan, pengguna dapat
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: “--feature-gates=SCTPSupport=true,...” needs to be in backtick

File: content/id/docs/concepts/services-networking/service.md [hunk 19]
```diff
@@ -0,0 +1,1065 @@
...
+PROXY TCP4 192.0.2.202 10.0.42.7 12345 7\r\n
+```
+yang kemudian diikuti data dari klien.
+
+### SCTP
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Kubernetes memberikan dukungan bagi SCTP sebagai _value_ dari _definition_ yang ada pada 
+_Service_, `Endpoints`, _NetworkPolicy_ dan `Pod` sebagai fitur _alpha_. Untuk mengaktifkan fitur ini,
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: _NetworkPolicy_  needs to be in backtick

File: content/id/docs/concepts/services-networking/service.md [hunk 20]
```diff
@@ -0,0 +1,1065 @@
...
+        # The name of the Amazon S3 bucket where the access logs are stored
+        service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-prefix: "my-bucket-prefix/prod"
+        # The logical hierarchy you created for your Amazon S3 bucket, for example _my-bucket-prefix/prod_
+```
+
+#### Mekanisme _Draining_ Koneksi pada AWS
+
+Mekanisme _draining_ untuk ELB klasik dapat dilakukan dengan menggunakan anotasi 
+`service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled` serta mengatur 
+_value_-nya menjadi _"true"_. Anotasi
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: nit: true needs to be in backtick

File: content/id/docs/concepts/services-networking/service.md [hunk 21]
```diff
@@ -0,0 +1,1065 @@
...
+Ketika sebuah `Pod` dijalankan pada *node*, _kubelet_ menambahkan seperangkat variabel _environment_ 
+untuk setiap `Service` yang aktif. _Environment_ yang didukung adalah [Docker links compatible](https://docs.docker.com/userguide/dockerlinks/) variabel (perhatikan 
+[makeLinkVariables](http://releases.k8s.io/{{< param "githubbranch" >}}/pkg/kubelet/envvars/envvars.go#L49)) 
+dan variabel `{SVCNAME}_SERVICE_HOST` dan `{SVCNAME}_SERVICE_PORT`, dinama nama `Service` akan diubah 
+menjadi huruf kapital dan tanda _minus_  akan diubah menjadi _underscore_.
+
+Sebagai contoh, `Service` `"redis-master"` yang mengekspos _port_ TCP 6379 serta _alamat_
+_cluster IP_ _10.0.0.11_ akan memiliki _environment_ sebagai berikut:
+
+```shell
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: below codeblock format is broken https://deploy-preview-14117--kubernetes-io-master-staging.netlify.com/id/docs/concepts/services-networking/service/#variabel-environment

probably you have remove backticks inside the codeblock

File: content/id/docs/concepts/services-networking/service.md [hunk 22]
```diff
@@ -0,0 +1,1065 @@
...
+REDIS_MASTER_PORT=tcp://10.0.0.11:6379
+REDIS_MASTER_PORT_6379_TCP=tcp://10.0.0.11:6379
+REDIS_MASTER_PORT_6379`TCP`PROTO=tcp
+REDIS_MASTER_PORT_6379`TCP`PORT=6379
+REDIS_MASTER_PORT_6379`TCP`ADDR=10.0.0.11
+```
+
+Hal ini merupakan kebutuhan yang urutannya harus diperhatikan - `Service` apa pun yang 
+akan diakses oleh sebuah `Pod` harus dibuat sebelum `Pod` tersebut dibuat, 
+jika tidak _environment_ _variabel_ tidak akan diinisiasi.
```
Comment (girikuncoro) [inline content/id/docs/concepts/services-networking/service.md]: nit: variabel environment, not environment variabel

**Pull request merged**